contributing_to_docs/doc_guidelines.adoc

```diff
@@ -392,6 +392,18 @@ In the example, `cloud-provider-first` and `cloud-provider` are not defined by t
 For more information about creating text snippets, see the
 link:https://redhat-documentation.github.io/modular-docs/#using-text-snippets[Red Hat modular docs reference guide]
 
+[id="Auto-generated-content"]
+== Auto-generated content
+
+* The following content is auto-generated in each release and should not be manually edited:
+** The {product-title} CLI (`oc`) command references `modules/oc-by-example-content.adoc` and `modules/oc-adm-by-example-content.adoc`.
```

Comment:
```suggestion
** The OpenShift CLI (`oc`) command references `modules/oc-by-example-content.adoc` and `modules/oc-adm-by-example-content.adoc`.
```

Just a reminder not to use the "Commit suggestion" button for this :)

Reply:
ok noted :)